spec/opera/operation/result_spec.rb

```diff
@@ -19,11 +19,10 @@ module Opera
     describe '#failures' do
       before do
         subject.add_error(:foo1, :bar1)
-        subject.add_exception(:foo2, :bar2)
       end
 
-      it 'returns errors and exceptions combined' do
-        expect(subject.failures).to eq(foo1: [:bar1], 'foo2' => :bar2)
+      it 'returns errors combined' do
```

Comment:
We're not combining anymore :)

```suggestion
      it 'returns errors' do
```